aerosol_dynamics/physical_parameters.py

```diff
@@ -2,28 +2,32 @@
 Centralized location for physical parameters.
 """
 
-from . import ureg
+from . import u
 
 # air viscosity in kg/m-s
-MEDIUM_VISCOSITY = 18.27e-6 * ureg.kg / ureg.m / ureg.s
+MEDIUM_VISCOSITY = 18.27e-6 * u.kg / u.m / u.s
 
 # mean free path of air in m
-MEAN_FREE_PATH_AIR = 65e-9 * ureg.m
+MEAN_FREE_PATH_AIR = 65e-9 * u.m
 
 # Boltzmann's constant in m^2 kg s^-2 K^-1
-BOLTZMANN_CONSTANT = 1.380649e-21 * ureg.m ** 2 * ureg.kg / ureg.s / ureg.K
+BOLTZMANN_CONSTANT = 1.380649e-21 * u.m ** 2 * u.kg / u.s / u.K

```

Comment:
This need to be `e-23`. Other than that, I approve of this pr, stellar work 🥇 

Thank you both!!!

Reply:
Thanks for catching that (again!)